eidas_node/tests/proxy_service/test_cznia_views.py

```diff
@@ -0,0 +1,92 @@
+from base64 import b64encode
+from copy import deepcopy
+from typing import BinaryIO, cast
+from unittest.mock import patch
+
+from django.test import RequestFactory, SimpleTestCase, override_settings
+from django.urls import reverse
+
+from eidas_node.models import LightResponse, Status
+from eidas_node.proxy_service.cznia.views import CzNiaResponseView
+from eidas_node.proxy_service.views import IdentityProviderResponseView
+from eidas_node.tests.proxy_service.test_views import DATA_DIR
+from eidas_node.tests.test_models import LIGHT_RESPONSE_DICT
+from eidas_node.tests.test_storage import IgniteMockMixin
+from eidas_node.utils import dump_xml
+
+
+@override_settings(ROOT_URLCONF='eidas_node.proxy_service.cznia.urls')
+class TestCzNiaResponseView(IgniteMockMixin, SimpleTestCase):
+    def setUp(self):
+        self.factory = RequestFactory()
+        self.url = reverse('identity-provider-response')
+        # self.addCleanup(self.mock_ignite_cache())
```

Comment:
If the cache is not needed, then this should be removed.

Reply:
I also removed `IgniteMockMixin` from parent classes.